service/controller/v2/resource/deployment/resource.go

```diff
@@ -83,8 +83,9 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	// Deleting the resource group will take care about cleaning
 	// deployments.
 	if key.IsDeleted(customObject) {
-		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling deployment deletion: deleted with the resource group")
+		r.logger.LogCtx(ctx, "level", "debug", "message", "redirecting responsibility of deletion of deployments to resource group termination")
```

Comment:
It's good that the message is improved but this is really long and I think will be hard to read in logs. What about something like this?

`deletion of deployments is handled by resource group termination`

Reply:
The important part here is the redirecting. I made it to this. 

```
deletion of deployments is redirected to resource group termination
```

Reply:
Thanks, I like that.